ztools/scada/opcua/helper.go

```diff
@@ -0,0 +1,96 @@
+package opcua
+
+import (
+	"encoding/hex"
+	"strconv"
+)
+
+
+func Checkerr(err error) error{
+	if err != nil {
+	    return err
+	}
+	return nil
+}
+
+// function to convert and parse little endian to integer32+
+func ParseLittleEndian(s string) int{
+	tmp := s[6:8] + s[4:6] + s[2:4] + s[0:2]
+	length64, _ := strconv.ParseInt(tmp, 16, 32)
+	length := int(length64)
+	return length
+}
+
+// function to parse pascal strings used in opcua, pascal string are in form: [4 byte length ULInt32|actual string]
+func Process_pascal_string(anchor int,s string) (string,int){
```

Comment:
This function should be named `ProcessPascalString`, to match Golang convention (no underscores, camel case).